lib/sqlalchemy/sql/schema.py

```diff
@@ -2766,9 +2766,11 @@ class ForeignKey(DialectKWArgs, SchemaItem):
 
     _table_column: Optional[Column[Any]]
 
+    _colspec: Union[str, Column[Any], Callable[[], _DDLColumnArgument]]
+
     def __init__(
         self,
```

Comment:
**Michael Bayer** (zzzeek) wrote:

wrong

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4831